API.md

```diff
@@ -27,6 +27,29 @@ drawing | boolean | The ability to draw and delete features - default: `true`
 interactive | boolean | Keep all features permanently in edit mode - default: `false`
 keybindings | boolean | Keyboard shortcuts for drawing - default: `true`
 controls | Object | drawable shapes - default `{ marker: true, line: true, shape: true, square: true }`
+styles | Object | Add a style with any of the following properties: `gl-draw-polygon`, `gl-draw-polygon-stroke`, `gl-draw-line`, `gl-draw-point`, `gl-drawing-line`, `gl-drawing-polygon`, `gl-drawing-polygon-stroke`, `gl-drawing-points`, `gl-edit-line`, `gl-edit-polygon`, `gl-edit-polygon-stroke`, `gl-edit-point`, `gl-edit-point-mid`. The property should be an object with either the `layout` and/or `paint` properties as specified in the [Mapbox GL Style Reference](https://www.mapbox.com/mapbox-gl-style-spec/). It will overwrite the corresponding default  styles found in [`src/theme/`](https://github.com/mapbox/mapbox-gl-draw/tree/master/src/theme).
```

Comment:
can you please make the properties into a bulleted list for the sake of readability?